gbdxtools/images/meta.py

```diff
@@ -122,6 +123,21 @@ def read(self, bands=None):
             arr = self[bands, ...]
         return arr.compute(get=threaded_get)
 
+    def randwindow(self, window_shape):
+        row = random.randrange(window_shape[0], self.shape[1])
+        col = random.randrange(window_shape[1], self.shape[2])
+        return (row-window_shape[0],col-window_shape[0], row, col)
+
+    def iterwindows(self, count=64, window_shape=(256, 256)):
```

Comment:
this is all cool, but also seems like we should have another method that iters over each image window/chunk. Like a helper that yeilds every single one. Seems like that be useful and simple to do right?